src/reducer/admin/index.js

```diff
@@ -1,12 +1,13 @@
 import { combineReducers } from 'redux';
-import resourceReducer from './resource';
+import { createSelector } from 'reselect';
+import resourceReducer, * as resourceSelectors from './resource';
```

Comment:
please don't import `*`. Instead, explicitly list what you want to import:

```js
import resourceReducer, { getData, getRecord, getRecordsById }  from './resource';
```

To be repeated every time you `import *` in this PR